bip-emilengler-terminology-for-payment-recipient-identifiers.mediawiki

```diff
@@ -0,0 +1,56 @@
+<pre>
+  BIP: UNDEFINED
+  Title: Terminology for payment recipient identifiers
+  Author: Emil Engler <me@emilengler.com>
+          MarcoFalke <falke.marco@gmail.com>
+          Luke Dashjr <luke+bip@dashjr.org>
+  Status: Draft
+  Type: Informational
+  Created: 2019-10-17
+  License: CC0
+</pre>
+
+==Abstract==
+This BIP proposes a new term for 'address'
+
+==Specification==
+The new term is:
+''Bitcoin'' '''Invoice''' ''Address''
+
+The ''Bitcoin'' and ''Address'' parts are optional.
+The address suffix should only be used as a transitional step.
+
+A ''Bitcoin'' Invoice ''Address'' is a string of characters that can be used to indicate the intended recipient and purpose of a transaction.
+
+==Motivation==
+Bitcoin addresses are intended to be only used '''once''' and you should generate a new one for every new incoming payment.
```

Comment:
Shouldn't this sentence start with `Bitcoin invoice addresses…`? :grin:

Reply:
It describes a problem of the past and currently the present so I think it will be fine there. :P